version-control-login.rst

```diff
@@ -0,0 +1,43 @@
+.. post:: May 9, 2018
+   :tags: feature, new-user, versioncontrol
+   :author: David
+   :location: SAN
+
+
+Version Control Log in
```

Comment:
I'd say that the title should reflect the "social" part of the version control. I mean, you don't log in with git, but with github.

> Social Version Control Log in

Maybe?